scripts/rmc

```diff
@@ -27,6 +27,11 @@ def main():
     parser.add_argument("--visualize", action="store_true")
     parser.add_argument("--srcdir", default=".")
     parser.add_argument("--wkdir", default=".")
+    parser.add_argument("--no-default-checks", action="store_true", help="Disable all default checks")
+    parser.add_argument("--no-memory-safety-checks", action="store_true", help="Disable default memory safety checks")
```

Comment:
Do we want a way to selectively add these back in?

Reply:
The user can always add them back in with `rmc --no-memory-safety-checks file.rs -- <cbmc-flags>`